### PR TITLE
Stricter typing for timeoutPromise()

### PR DESCRIPTION
**Pull request by JonathanDCohen**

### CODE REVIEW

**Comment by alalamav** [comment]
Please update the call sites of `timeoutPromise` with the appropriate type.

File: src/www/app/util.ts [hunk 1]
```diff
@@ -12,13 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as errors from '../model/errors';
+import * as errors from "../model/errors";
```
Comment (alalamav) [inline src/www/app/util.ts]: nit (prettier fail): use single quotes here and in the next line.

**Comment by JonathanDCohen** [comment]
Callsites don't need to be updated, the type parameter is deduced by `tsc`

File: src/www/app/util.ts (hunk 1 above)
Comment (JonathanDCohen) [reply src/www/app/util.ts]: Done in ae44e1e

**Comment by JonathanDCohen** [comment]
Typically I like to keep generic parameter type deduction where possible as it makes future refactors easier.

**Pull request merged**